packages/modules/cra-template-desktop-viewer/template/src/backend/main.ts

```diff
@@ -0,0 +1,69 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Logger, LogLevel } from "@bentley/bentleyjs-core";
+import {
+  ElectronHost,
+  ElectronHostOptions,
+} from "@bentley/electron-manager/lib/ElectronBackend";
+import { Presentation } from "@bentley/presentation-backend";
+import * as path from "path";
+
+import { AppLoggerCategory } from "../common/LoggerCategory";
+import { viewerRpcs } from "../common/ViewerConfig";
+import { appInfo, getAppEnvVar } from "./AppInfo";
+import ViewerHandler from "./ViewerHandler";
+
+require("dotenv-flow").config(); // eslint-disable-line @typescript-eslint/no-var-requires
```

Comment:
Does `dotenv-flow` handle expansion/tempalting as well? If so, I'd be interested in moving to use it in the iModel.js repo.

Reply:
I don't believe so. You'd need dotenv-expand I believe as well. This just adds predefined support for .local

Reply:
gotcha, okay